life/updated/01-preface.adoc

```diff
@@ -6,81 +6,80 @@ ____
 
 == Preface
 
-As the memory of the just is blessed, so there is a justice due to their memories,
-who walked in the path of the just, which is as a shining light,
-and are the blessed who die in the Lord, who rest from their labors,
-and their works follow them to their everlasting reward in the kingdom of heaven;
-as well as for the example and admonition of those they leave behind.
-This is either by publishing their own memoirs of their lives, when they leave any;
+As the memory of the just is blessed, so there is a justice due to the memories
+of those who walked in the path of the just, which path is as a shining light.
+Blessed are they who die in the Lord, who rest from their labors,
+and their works follow them to their everlasting reward in the kingdom of heaven.
+And blessed are they as well for the example and admonition they leave behind,
+which is either by the publishing of their memoirs,
 or where that is lacking,
 by commemorating them according to the example of holy Scripture.
 The following relation of her labors and sufferings for Christ,
 written by an early disciple in this Gospel-day,
 renders it unnecessary to say much of the author;
-her own book being a sufficient memorial of her, and what she was;
-only as a necessary introduction to her own account,
-I have this to premise concerning her.
+her own book being a sufficient memorial of her, and what she was.
+I have only the following necessary introduction to leave on her account.
 
-I was acquainted with her many jeers, she being my country-woman;
+I was acquainted with her many years, she being my country-woman,
 and this testimony lives in my heart concerning her,
 that she was a religiously exercised woman, always seeking the best things,
-and through the goodness of God, found him whom her soul loved.
-She underwent many deep exercises inwardly and outwardly, by contrary spirits,
-who opposed the work of the Lord, and his servants and handmaids.
-She bore a faithful testimony, as the Lord laid it upon her, and made way for her,
-being a true mourner in Zion, for the abominations of the times.
-She had a solid savory testimony for the truth, which she received early,
-and was faithful and diligent in, according to her gift,
-and was well esteemed for her service by the honest-hearted in and about Bristol,
+and through the goodness of God found Him Whom her soul loved.
```

Comment:
I would maybe say, "....and through the goodness fo God, **she** found Him whom her soul loved. I think I would kill the capitalization of Whom here too, because it follows directly after Him.